packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/data_source/pin_biometrics_auth_data_source.dart

```diff
@@ -4,7 +4,10 @@ import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
 import '../models/biometrics_authentication_type.dart';
 
 class PinBiometricsAuthDataSource extends BiometricsAuthDataSource {
-  PinBiometricsAuthDataSource({required super.localAuthentication});
+  PinBiometricsAuthDataSource({required this.localAuthentication});
+
+  @override
```

Comment:
Remove this `@override` because it is not part of the interface anymore.

Reply:
It is removed